src/components/chip/chip.4.example

```diff
@@ -1,7 +1,24 @@
-html!{
-    <>
-        <Chip text="Foo" onclose={Callback::from(|_|{})} />
-        <Chip text="Bar" onclose={Callback::from(|_|{})} />
-        <Chip text="Baz" onclose={Callback::from(|_|{})} />
-    </>
-}
\ No newline at end of file
+{
+    #[function_component(ClosableChips)]
+    fn closable_chips() -> Html {
+        use yew::prelude::*;
+
+        let options = use_state(|| vec!["Foo", "Bar", "Baz"]);
+        let close = |idx| {
+            let options = options.clone();
+            Callback::from(move |_| {
```

Comment:
I know we are still using `Callback::from` in some cases (too many). But moving forward I would prefer to use `use_callback`, as it provides a stable instance, and doesn't cause re-renders.

Reply:
Ah, damn. I missed that! You're right, I am sorry.